lib/meadow_web/resolvers/ingest.ex

```diff
@@ -123,7 +123,18 @@ defmodule MeadowWeb.Resolvers.Ingest do
          details: ChangesetErrors.error_details(changeset)}
 
       {:ok, ingest_sheet} ->
-        {:ok, ingest_sheet}
+        Validator.validate(ingest_sheet.id)
```

Comment:
The only problem I see here is that the mutation won't return until validation completes and ingest starts. This might cause problems (or timeouts) for large ingest sheets.

Reply:
Good call. Any suggestions on how to avoid those problems? I feel like that's going to be tricky to solve with how we do validation and approval currently.

Reply:
Maybe move the auto-approve code out of the resolver and into the validator itself? Maybe even pass an `auto_approve` parameter to the validator?

Reply:
Thanks for the assist @mbklein and @kdid!